tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -31,25 +34,63 @@ class InactiveUser(Exception):
     pass
 
 
+class UnverifiedEmail(Exception):
```

Comment:
Is this just for easier traceability for exceptions or did you want to extend functionality here in the future?

Reply:
For easier traceability for exceptions and improved test coverage